src/controllers/commands/CommandController.cpp

```diff
@@ -482,6 +482,21 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         QString userName = words[1];
         stripUserName(userName);
+
+        if (words.size() > 2)
+        {
+            QString channelName = words[2];
+            stripChannelName(channelName);
+
+            ChannelPtr channelTemp =
+                getApp()->twitch2->getChannelOrEmpty(channelName);
+
+            if (channelTemp->getType() != Channel::Type::None)
```

Comment:
```suggestion
            if (channelTemp->isEmpty())
```
I believe that using `isEmpty()` is better because it's clearer to understand what this code does. :)